src/Microsoft.AspNetCore.Http.Features/IHttpRequestFeature.cs

```diff
@@ -49,6 +49,11 @@ public interface IHttpRequestFeature
         string QueryString { get; set; }
 
         /// <summary>
+        /// The request target as it was sent in the HTTP request.
+        /// </summary>
+        string RawTarget { get; set; }
```

Comment:
That's what do comments are for :-). Link to the RFC section.

Besides, most people don't even know what feature interfaces are.